ddpui/dbt_automation/operations/mergeoperations.py

```
@@ -141,3 +140,98 @@ def merge_operations(
+        elif operation["type"] == "generic":
+            op_select_statement, out_cols = generic_function_dbt_sql(operation["config"], warehouse)
+        elif operation["type"] == "rawsql":
+            op_select_statement, out_cols = raw_generic_dbt_sql(operation["config"], warehouse)
+
+        output_cols = out_cols
+
+        cte_sql = f" , {operation['as_cte']} as (\n"
+        if cte_counter == 0:
+            cte_sql = f"WITH {operation['as_cte']} as (\n"
+        cte_sql += op_select_statement
+        cte_sql += f")"
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)